features/upgrade/api_auth/upgrade.feature

```diff
@@ -0,0 +1,56 @@
+Feature: apiserver and auth related upgrade check
+  # @author kewang@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Default RBAC role, rolebinding, clusterrole and clusterrolebinding without any missing after upgraded - prepare
+    When I run the :get admin command with:
+      | resource | clusterroles.rbac |
+      | o        | yaml              |
+    Then the output is parsed as YAML
```

Comment:
Not necessary step.

Reply:
@liangxia You mean line 9 is not necessary?
 Then the output is parsed as YAML

Reply:
Yes. I didn't notice it.

Reply:
Removed the unnecessary steps for yaml file format checking